src/dotty/tools/dotc/core/Comments.scala

```diff
@@ -125,7 +129,7 @@ object Comments {
               val newName = (tree.name.show + "$" + codePos + "$doc").toTermName
               untpd.DefDef(newName, tree.tparams, tree.vparamss, tree.tpt, tree.rhs)
             case _ =>
-              ctx.error("proper definition was not found in `@usecase`", codePos)
+              ctx.error(ProperDefinitionNotFound(), codePos)
```

Comment:
To get a good sense of what's going on here for the error message's explanation there are a couple of things you ( or the user) need(s) to know:

1. What are `@usecase`?
2. Why do we care that we're getting an `untpd.DefDef` here?